Contribute/dotnet/dotnet-contribute.md

```diff
@@ -67,6 +67,60 @@ For **code snippets**, create a subfolder called **snippets** inside the folder
 
 Code snippets are small, focused examples of code that demonstrate the concepts covered in an article. Larger programs intended for download and exploration should be located in the [dotnet/samples](https://github.com/dotnet/samples) repository. Full samples are covered in the section on [Contributing to samples](#contribute-to-samples).
 
+### Contribute .NET analyzer docs
+
+.NET compiler platform (Roslyn) analyzers inspect your C# or Visual Basic code for code quality and code style issues. Starting in .NET 5.0, these analyzers are [included with the .NET SDK](/dotnet/fundamentals/code-analysis/overview).
+
+- [Code quality analysis ("CAxxxx" rules)](/dotnet/fundamentals/code-analysis/overview#code-quality-analysis):
+  - Implemented [here](https://github.com/dotnet/roslyn-analyzers/tree/master/src/NetAnalyzers) in `dotnet/roslyn-analyzers` repo.
+  - Documented [here](https://github.com/dotnet/docs/blob/master/docs/fundamentals/code-analysis/quality-rules) in the `dotnet/docs` repo. See [Contribute docs for 'CAxxxx' rules](#contribute-docs-for-caxxxx-rules).
+- [Code style analysis ("IDExxxx" rules)](/dotnet/fundamentals/code-analysis/overview#code-style-analysis):
+  - Implemented [here](https://github.com/dotnet/roslyn/tree/master/src/Analyzers) in `dotnet/roslyn` repo.
+  - Documented [here](https://github.com/dotnet/docs/blob/master/docs/fundamentals/code-analysis/style-rules) in the `dotnet/docs` repo. See [Contribute docs for 'IDExxxx' rules](#contribute-docs-for-idexxxx-rules).
+
+#### Contribute docs for 'CAxxxx' rules
+
+Please follow the below steps to contribute documentation for CA rules to the [dotnet/docs](https://github.com/dotnet/docs) repo:
+
+1. Determine `Rule ID` and `Category`: Ensure that you know the 'CAxxxx' rule ID and category for the rule to be documented. This means either your CA analyzer has been merged into [dotnet/roslyn-analyzers](https://github.com/dotnet/roslyn-analyzers) repo or you have an open PR with an approved ID and category that has been assigned to the rule.
+2. Add an entry for the rule to following tables:
+  1. In [table of contents](https://github.com/dotnet/docs/blob/master/docs/fundamentals/toc.yml) file under the rule category list. For example, for a CA rule with `Performance` category, search for the term `- name: Performance rules` in the file and add an entry to the list. If none exists, create a new category list under `- name: Code quality rules`.
```

Comment:
Since these steps aren't an ordered procedure, they should be bulleted instead of numbered.